main.cpp

```diff
@@ -326,6 +578,29 @@ Thing 10) Elevator
     2) Open and close
     3) Remember floor chosen by user
  */
+struct Elevator
+{
+
+    //Make new Cabin
+    Cabin lobbyElevatorCabinOne;
+    //Make new Door
+    Door lobbyElevatorDoor_one, lobbyElevatorDoor_two;
+    //Make new Panel
+    Panel lobbyElevatorPanel;
+    //Make new Control
+    Control lobbyElevatorControl;
+    //Make new Motor
+    Motor lobbyElevatorMotor;
+
+    //Move up and down
+    void elevatorAscend(Control requestedFloor, Motor elevatorAcceleration);
+    //Open and close
+    void openDoor(Door doorClosureStatus, Door speedOfDoorMovement);
+    //Remember floor chosen by user
+    void storeRequestedFloor(Control requestedFloor);
```

Comment:
All of these function parameters are being incorrectly used here.. 
It's very common for people new to C++ to think that they need to pass in the member variable into the function so that they can use it.
That is what it appears you are doing here.
You do not need to do that. When you call these member functions, you have access to these member variables because they are part of the elevator class instance that you are calling these number functions on.
```
Elevator elevator;
elevator.openDoor(); //I can use `leftDoor` and `rightDoor` inside of this function without doing anything special
```
That looks like this:
```
void Elevator::openDoor()
{
    leftDoor.openDoor();
    rightDoor.openDoor();
}
```

Please read the note that I put in the instructions where I discussed the example of setting the manager for a team. 
It is very important that you understand this concept and why you do not need to pass in instances of these classes into these member functions.

As an example, consider your ascend function. You are requesting a floor. That can be defined with an integer. You would then set the request the floor in the control object inside of this function. Same goes for the Acceleration Rate that you want.  You would set the acceleration property of the motor member variable.

```
void Elevator::ascend(int floor, float accelerationRate)
{
    control.setDestinationFloor( floor );
    motor.setAccelerationRate( accelerationRate );
}
```
- [ ] revise

Reply:
This is very helpful. I understand thie concept, but still find the idea difficult. I just need more practice.